### DISCUSSION TITLE
connection over https

### DISCUSSION THREAD

**Discussion by bennysweb** [solectrus/solectrus#2296] [Q&A]
Hi, 
when I set the FORCE_SSL entry in the .env file to true in the I can't open the solectrus dashboard anymore (now over https://192.168...., formerly over http://192.168.... it works poerfectly). Clearing browser cache and reset the history didn't help. The log shows the following: 
_"HTTP parse error, malformed request: #<Puma::HttpParserError: Invalid HTTP format, parsing fails. Are you trying to open an SSL connection to a non-SSL Puma?>"_
Is there anything else what I should do or isn't it possible to change the redirect after installing?
Best regards,
Benny

**Comment by ledermann** [comment]
Sorry for the trouble, but the option `FORCE_SSL` is misleading. What it does:

- It forces all access to the app over SSL by doing a permanent redirect
- It uses Strict-Transport-Security (HSTS)
- It uses secure cookies

It does **NOT** handle certificate or cipher stuff.

After enabling it accidentally, you are now caught in the HSTS trap: `https` does not work, but `http` is still always redirected to `https` because the browser does not even try `http` anymore.

To fix:

- Change `FORCE_SSL` back to `false` and restart the app container
- Open SOLECTRUS in another browser to check if it runs fine
- Clear HSTS cache for `192.168...` in your regular browser, which can be annoying. Browser DevTools may be helpful.

References:
- https://github.com/solectrus/hosting/issues/21

**Reply by bennysweb** [reply]
Thanks for sorting this out. Now it works with the internally Synology Reverse Proxy solution and I can reach my solectrus domain via https. 
Benny
